client/src/services/allocations.js

```diff
@@ -0,0 +1,28 @@
+import store from 'store';
+import { API_URL } from '../constants';
+
+export const createPhaseAllocation = async (allocationJson) => {
```

Comment:
I know it's a convention established in other files, but the fact that a parameter called `allocationJson` isn't actually JSON bothers me. I'd suggest renaming it to something like `allocationData`, to better hint at its type.

Reply:
renamed to payload!

Reply:
Much better!